vyper/parser/arg_clamps.py

```diff
@@ -68,31 +68,44 @@ def make_arg_clamper(datapos, mempos, typ, is_init=False):
     # Lists: recurse
     elif isinstance(typ, ListType):
         if typ.count > 5 or (type(datapos) is list and type(mempos) is list):
-            subtype_size = get_size_of_type(typ.subtype)
-            i_incr = subtype_size * 32
+            # find ultimate base type
+            subtype = typ.subtype
+            while hasattr(subtype, "subtype"):
+                subtype = subtype.subtype
 
-            mem_to = subtype_size * 32 * (typ.count - 1)
+            # make arg clamper for the base type
+            offset = 288
```

Comment:
hello magical number

Reply:
Hrmm, good point.  I had to look up what it was, I should probably also make it easier for future devs.